prism/prism.js

```diff
@@ -22,7 +22,7 @@
  SOFTWARE.
  */
 /* PrismJS 1.23.0
-https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript+json+rego */
+https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript+json+rego&plugins=line-numbers */
```

Comment:
Do we need the line number plugin at all? I removed it locally and looked like the numbering still worked correctly.

Reply:
I guess I did not try it without. Prism is supposed to support it out of the box but with the editor we're using it still required the manual mapping. Good catch!